config.yml

```
@@ -33,7 +33,14 @@ genesis:
     staking:
       params:
         bond_denom: uojo
+  consensus:
+    params:
+        abci:
+          vote_extensions_enable_height: "2"
   chain_id: ojo-testnet
 validators:
 - name: alice
   bonded: 33500000000000uojo
+  app:
+    pricefeeder:
+      config_path: "PATH TO PRICE FEEDER CONFIG HERE"
```

Can we set this to a default in the repo?
```suggestion
      config_path: "PATH TO PRICE FEEDER CONFIG HERE"
```

---

Updated to relative path